massa-bootstrap/src/listener.rs

```diff
@@ -111,6 +111,17 @@ impl BootstrapTcpListener {
             }
         }
 
+        // We need to have an accept() error with WouldBlock, otherwise polling may not raise any new events.
+        // See https://users.rust-lang.org/t/why-mio-poll-only-receives-the-very-first-event/87501
```

Comment:
this discussion concludes that we need to implement exactly what is implemented above no ?

Reply:
No, the loop above does that for the std::net::TcpListener, we need it for the mio_server :)

Reply:
what happens if we call accept above on mio_server instead of server ?

Reply:
Then we retrieve mio::net::TcpStream instead of std ones, and I don't think we can easily convert from one to the other (although I didn't look into it much, there is a from_std method but not to_std)

Reply:
> what happens if we call accept above on mio_server instead of server ?

that return a TcpStream of mio, but we need a std TcpStream